openedx/frontend-app-course-authoring
# feat: allow dragging blocks across parents in outline

```yaml
EDX_PLATFORM_REPOSITORY: https://github.com/openedx/edx-platform.git
EDX_PLATFORM_VERSION: master

TUTOR_GROVE_WAFFLE_FLAGS:
  - name: contentstore.new_studio_mfe.use_new_course_outline_page
    everyone: true
```

**Description:**
This PR modifies drag-n-drop of elements in outline to support possibility of dragging elements across different parent elements.

**Test instructions:**
* Get devstack up and running using `make {lms,cms}-up`.
* Start npm devserver in this repo using `npm start`.
* Go to outline page of any course via cms and update port to 2001.
* Drag and drop sections, subsections and units and confirm that they can be dropped in other parent elements similar to legacy.
* PR also updates move up and down menu action to support moving across parents.

`Private-ref`: [BB-8634](https://tasks.opencraft.com/browse/BB-8634)

Closes: #851
